### DISCUSSION TITLE
Indicating when a quest was 'last surveyed'

### DISCUSSION THREAD

**Discussion by dbdean** [streetcomplete/StreetComplete#4536] [Ideas]
I was wondering what the communities thoughts might be on displaying an indicator on quests about when the information being asked was last updated?

I'm asking because a bunch of resurvey 'Does this crossing have tactile paving on both sides?' quests have just come up in my area, and I would be interested in having the app tell me when they were last surveyed when asking the question to help the user understand why they have appeared now.

If it was implemented, I guess I would see it as maybe adding some extra text maybe near the question ("last surveyed more than 6 months ago", "never surveyed before", etc.)

**Comment by Helium314** [comment]
If there is no `check_date:tactile_paving` or similar, such data would need to be extracted from the element history, which is currently not available in SC.

**Comment by matkoniecz** [comment]
> I would be interested in having the app tell me when they were last surveyed when asking the question to help the user understand why they have appeared now.

This sounds like extra info that is rather in expert/special interest, not needed to answer. Maybe providing link to edited objects or link to history of edited objects would be more generic and better working?
